preview/Win7Msix/Win7MSIXInstaller/InstallUI.cpp

```diff
@@ -464,17 +475,15 @@ BOOL ChangeText(HWND parentHWnd, std::wstring displayName, std::wstring messageT
     layoutRect.Y += 40;
     graphics.DrawString(messageText.c_str(), -1, &messageFont, layoutRect, &format, &blackBrush);
 
-    if (logoStream != nullptr)
-    {
-        // We shouldn't fail if the image can't be loaded, just don't show it.
-        g_image = Gdiplus::Image::FromStream(logoStream, FALSE);
-            
-    }
-        
-    if (g_image != nullptr)
-    {
-        Gdiplus::Status status = graphics.DrawImage(g_image, g_width - 200, 25);
-    }
+	if (logoStream != nullptr)
+	{
+		// We shouldn't fail if the image can't be loaded, just don't show it.
+		auto g_image = Gdiplus::Image::FromStream(logoStream, FALSE);

```

Comment:
>g_image [](start = 7, length = 7)

appreciate removing unnecessary globals; please change the name though, thanks!

Reply:
renamed!